torch/jit/__init__.py

```diff
@@ -1492,9 +1490,11 @@ def __getattr__(self, attr):
             try:
                 return ScriptModule.__getattr__(self, attr)
             except AttributeError:
-                orig = self.__dict__["_original"]()
-                if orig and self.__dict__["_initialized"]:
-                    return getattr(self.__dict__["_original"](), attr)
+                # unwrap the original
+                original_module = self.__dict__["_original"]()
+                if original_module and self.__dict__["_initialized"]:
```

Comment:
Why do we need the "initialized" check here?

Reply:
So the `__getattr__` only looks at the original model after `__init__` is done